public/examples/alpheios-integration/index.html

```diff
@@ -4,7 +4,7 @@
 <head>
     <meta charset="UTF-8">
     <title>Alpheios Text/Treebank Alignment Demo</title>
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/alpheios-components@beta/dist/style/style-components.css"/>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/alpheios-components@latest/dist/style/style-components.css"/>
```

Comment:
it's a good question. Ideally I would like to be able to commit to not doing breaking changes and `@latest` is a synonym for  `@stable` but I cannot guarantee we won't have to make breaking changes in the future.  The advantage of using the `@latest` tag is that sites will automatically get bug fixes without having to make any changes on their side. The disadvantage is that their sites might at some point in the future break.  Perhaps I should include instructions for both options and let users make their own decision about which way to go?

Reply:
That's a difficult choice. I tend to lean towards not breaking projects over delivering bug fixes because it seems like a lot of DH projects are made by students or with a grant and end up unmaintained after creation. But I don't know the particular pros and cons with the Alpheios Components library so I'll leave it up to you. I have no hesitations merging this PR either way.

Eventually I do think it would make sense to have an "advanced instruction" section or something similar that explains the tradeoffs, but I don't think there's any reason to rush it and put it in this PR.

Reply:
ok, then let's leave it as is, and I'll make an issue for myself so that I don't forget. thanks!